src/main/scala/tech/cryptonomic/conseil/tezos/ServiceOperations.scala

```diff
@@ -0,0 +1,63 @@
+package tech.cryptonomic.conseil.tezos
+
+import com.typesafe.config.Config
+
+import scala.concurrent.{ExecutionContext, Future}
+import scala.collection.JavaConverters._
+
+
+object ServiceOperations {
+
+  final case class Counts(blocks: Int, accounts: Int, operationGroups: Int, operations: Int, fees: Int)
+
+  final case class Network(name: String, displayName: String, platform: String, network: String)
+
+  final case class Entity(name: String, displayName: String, count: Int, network: String)
+
+  def getNetworks(config: Config): List[Network] = {
+    config.getObject("platforms").asScala.flatMap {
+      case (platform, strippedConf) =>
+        strippedConf.atKey(platform).getObject(platform).asScala.map {
+          case (network, _) =>
+            Network(network, network.capitalize, platform, network)
+        }.toList
+    }.toList
+  }
+
+  def getEntities(network: String)(implicit ec: ExecutionContext): Future[List[Entity]] = {
```

Comment:
Is it possible to get the name directly from the slick schema? Look into that, because then you can make Entity creation table agnostic, and you can just apply an entity creating function to every element of the counts tuple, rather than hardcoding a List.

Reply:
I tried to do it more agnostic, but not its not as good as I would like. Slick is didn't give me a lot to work with. changed it in the new PR.